src/main/java/com/cleanroommc/groovyscript/compat/mods/pyrotech/BrickOven.java

```diff
@@ -0,0 +1,108 @@
+package com.cleanroommc.groovyscript.compat.mods.pyrotech;
+
+import com.cleanroommc.groovyscript.api.GroovyLog;
+import com.cleanroommc.groovyscript.api.IIngredient;
+import com.cleanroommc.groovyscript.api.documentation.annotations.*;
+import com.cleanroommc.groovyscript.helper.ingredient.IngredientHelper;
+import com.cleanroommc.groovyscript.helper.recipe.AbstractRecipeBuilder;
+import com.cleanroommc.groovyscript.registry.ForgeRegistryWrapper;
+import com.codetaylor.mc.pyrotech.modules.tech.basic.ModuleTechBasic;
+import com.codetaylor.mc.pyrotech.modules.tech.basic.recipe.CampfireRecipe;
+import com.codetaylor.mc.pyrotech.modules.tech.machine.ModuleTechMachine;
+import com.codetaylor.mc.pyrotech.modules.tech.machine.recipe.BrickOvenRecipe;
+import com.codetaylor.mc.pyrotech.modules.tech.machine.recipe.StoneOvenRecipe;
+import net.minecraft.item.ItemStack;
+import org.jetbrains.annotations.Nullable;
+
+@RegistryDescription(admonition = {@Admonition(value = "groovyscript.wiki.pyrotech.oven.note0",
+                                               type = Admonition.Type.DANGER,
```

Comment:
i think this might be better as a type WARNING - additionally, if there is only a single entry, there is no need to include curly braces `{}` around the annotation